runtime-config-linux.md

```diff
@@ -17,7 +17,7 @@ The following parameters can be specified to setup namespaces:
     * **`uts`** the container will be able to have its own hostname and domain name
     * **`user`** the container will be able to remap user and group IDs from the host to local users and groups within the container
 
-* **`path`** *(string, optional)* - path to namespace file
+* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
```

Comment:
On Fri, Dec 18, 2015 at 11:02:55AM -0800, Michael Crosby wrote:
> > -* **`path`** *(string, optional)* - path to namespace file
> > +* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
>
> :disappointed: 
>
> It is the namespace of the runtime (not host) as you can nest things
> so this is not exactly correct to say.

If we prefer “runtime” to “host” for that distinction, I'm fine
changing that.  Ideally we'd land a glossary like #107 so we had a
place to define these terms.  For me:

* host namespace: a namespace you are in when you invoke the runtime
* host process: the runtime process invoked by the user
* container process: the process created by a clone call in the host
  process which will eventually execute the user-configured process.

Both the host and container processes are running runtime code
(although the container process eventually transitions to
user-configured code), so I find “runtime process”, “runtime
namespace”, etc. to be imprecise.

@mrunalp used the “host” phrasing in some of the earlier #231 commits
[1,2], but again, if “host” → “runtime” gets me to a consensus naming
scheme, then I'll reroll this commit.

[1]: https://github.com/opencontainers/specs/pull/231#discussion-diff-42823459R41
[2]: https://github.com/opencontainers/specs/pull/231#discussion-diff-42832434R49